Tests/Fixtures/Test.plist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kwridan adds this `Test.plist` to fixture for unit testing `readPlist` function